Code/Source/ROS2EditorModule.cpp

```diff
@@ -24,9 +25,7 @@ namespace ROS2
             // EditContext. This happens through the [MyComponent]::Reflect() function.
             m_descriptors.insert(
                 m_descriptors.end(),
-                {
-                    ROS2EditorSystemComponent::CreateDescriptor(),
-                });
+                { ROS2EditorSystemComponent::CreateDescriptor(), ROS2RobotImporterEditorSystemComponent::CreateDescriptor() });
```

Comment:
minor: Can you format (align) the descriptor list so its easier to read the list?

Reply:
See https://github.com/RobotecAI/o3de-ros2-gem/pull/121#discussion_r958545475